### PR TITLE
Adding spanish translation

### PR DESCRIPTION
**Pull request by giobeatle1794**

### CODE REVIEW

File: i18n/es.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "@metadata": {
+        "authors": [
+            "Aleksi Postari"
```
Comment (kulttuuri) [inline i18n/es.json]: You can change yourself as the author of this translation, if you wish :)

**Comment by kulttuuri** [comment]
Added a comment above. So, feel free to change yourself as the author of this translation, if you wish to :)

File: i18n/es.json (hunk 1 above)
Comment (giobeatle1794) [reply i18n/es.json]: Done. Now, it's ready to merge.

**Comment by kulttuuri** [comment]
Merged! Thank you :) 👍

File: i18n/es.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "discordnotifications-article-saved-minor-edits": "hizo una edición menor en",
+    "discordnotifications-article-saved-edit": "editó",
+    "discordnotifications-article-created": "📄 %s creó el artículo %s %s",
+    "discordnotifications-file-namespace": "Archivo",
+    "discordnotifications-article-deleted": "❌ %s borró el artículo %s Motivo: %s",
+    "discordnotifications-article-moved": "➡ %s movió el artícle %s a %s. Motivo: %s",
+    "discordnotifications-article-protected": "🔒 %s %s artículo %s. Motivo: %s",
+    "discordnotifications-article-protected-change": "cambió la protección del",
+    "discordnotifications-article-protected-remove": "quitó la protección del",
+    "discordnotifications-new-user": "👥 La cuenta de usuario %s fue creada %s",
```
Comment (fitojb) [inline i18n/es.json]: Don’t use passive voice; it’s not good style. Better use _pasiva refleja._

Comment (kulttuuri) [reply i18n/es.json]: @giobeatle1794 changes all good to go? :)

**Pull request merged**